src/android/org/apache/cordova/plugin/sentinel/modules/DevicesModule.kt

```diff
@@ -0,0 +1,182 @@
+package org.apache.cordova.plugin.sentinel.modules
+
+import com.qxdev.sentinel_sdk.SentinelProvider
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import org.apache.cordova.CallbackContext
+import org.apache.cordova.plugin.sentinel.interfaces.ModuleDelegate
+
+class DevicesModule : ModuleDelegate {
```

Comment:
Missing `executeAction` ? Also all functions should be private

Reply:
Done